### DISCUSSION TITLE
Can we use SuperSimpleTcp with android(Java)?

### DISCUSSION THREAD

**Discussion by ubaidkhanturk** [jchristn/SuperSimpleTcp#163] [Q&A] [answered]
I try, i create Server in C# with supersimpleTcp and create client in android(JAVA). Android Client successfully connect with server and send msgs from client to server easily but the issue is that when i send msg from Server to client(Android) it did not send until i stop debugging. When i stop debbuging Server the msg will recieve to client.
Im stuck here i dont know what to do.. Can anyone help me with send message from server to client(Android JAVA).

**Comment by jchristn** [comment, marked as answer]
Hi @ubaidkhanturk were you able to get this working?  There should not be any incompatibility across operating systems or languages with SuperSimpleTcp (it doesn't apply any framing or anything).  Did everything work with the provided SuperSimpleTcp client with the server you built?

**Reply by ubaidkhanturk** [reply]
no.. i did'nt achieve this task. but i find my own way.. in this scenario client(Android) to server(windows) communication was wasy but it's difficult to send message from server to client without restarting server. so when i send message from server to client, every time i dispose my server and restart it again.. this is the only way to send msg from server to client
